blog/_posts/2021-04-15-affils-update.md

```diff
@@ -0,0 +1,137 @@
+
+**Historically Black Colleges and Universities (HBCUs)**
+
+As of last year, we already had assigned identifiers for a number of Historically Black Colleges and Universities (HBCUs) in the United States, particularly those with historically strong programs in science and engineering (most notably: Hampton University in Virginia, and Howard University in Washington, DC.)  During the past year, we have expanded our coverage to include all federally-recognized HBCUs represented in the ADS literature.  The list of HBCUs and their affiliation IDs can be accessed [here](https://github.com/adsabs/CanonicalAffiliations/blob/master/historically_black_colleges_universities.tsv).  Note that we have not assigned IDs to the entire list of federally designated schools; we excluded those not represented in literature and whose focus mostly lies outside the scope of the ADS, including business, law, and divinity schools.
+
+**Tribal Colleges and Universities (TCUs)**
+
+We have also added affiliation IDs for a partial list of Tribal Colleges and Universities (TCUs), primarily those in the United States that are currently represented in the ADS Literature.  The list of identified TCUs can be accessed [here](https://github.com/adsabs/CanonicalAffiliations/blob/master/tribal_colleges_universities.tsv); note that this list includes only those institutions designated as tribally-controlled, and does not include those public or private non-Tribal institutions having a significant Native American population; some of those schools are present in the list as public universities, however, mainly those that have science and engineering programs.
+
+There are comparatively few publications by authors affiliated with these colleges: Salish Kootenai College (Montana), Northwest Indian College (Pacific NW), Din&eacute; College (Arizona) and Fond du Lac Tribal College (Minnesota) have the most publications. Papers written by TCU authors and coauthors in the ADS primarily focus on topics in Environmental Science and Science Education, which are not currently part of our core literature.  These institutions may become more frequently represented if our scope widens to include more related sciences (e.g. planetary and allied literature).  We also want to ensure that their contributions are noted in existing literature, and that their contributions will be more easily tracked if these institutions increase their participation in the ADS core fields of astronomy and physics.
```

Comment:
The colon in the first sentence makes me think the four colleges listed have few publications, but the end of the sentence makes it sound like those 4 have the most. Can you clarify? If it's that these are the most represented, changing the colon to a period or semi-colon is all that's needed (probably a period - there are a lot of semi-colons in this subsection already).